pcdsdevices/valve.py

```diff
@@ -237,32 +237,38 @@ class VGC(VRC):
 
 class VFS(Device):
     """Class for Fast Shutter Valve."""
+    valve_position = Cpt(EpicsSignalRO, ':POS_STATE_RBV', kind='normal',
+                         doc='Ex: OPEN, CLOSED, MOVING, INVALID, OPEN_F')
+    vfs_state = Cpt(EpicsSignalRO, ':STATE_RBV', kind='normal',
+                    doc='Fast Shutter Current State')
     request_close = Cpt(EpicsSignalWithRBV, ':CLS_SW', kind='normal',
                         doc=('Request Fast Shutter to Close. When both close'
                              'and open are requested, VFS will close.'))
-    request_open = Cpt(EpicsSignalWithRBV, ':OPEN_SW', kind='normal',
+    request_open = Cpt(EpicsSignalWithRBV, ':OPN_SW', kind='normal',
                        doc=('Request Fast Shutter to Open. When both close and'
                             'open are requested, VFS will close.'))
-    reset_vacuum_fault = Cpt(EpicsSignalWithRBV, ':FAULT_RESET', kind='normal',
+    reset_vacuum_fault = Cpt(EpicsSignalWithRBV, ':ALM_RST', kind='normal',
                              doc=('Reset Fast Shutter Vacuum Faults: fast'
                                   'sensor triggered, fast sensor turned off.'
                                   'To open VFS, this needs to be reset to TRUE'
                                   'after a vacuum event.'))
-    override_mode = Cpt(EpicsSignalWithRBV, ':VFS_OVRD_ON', kind='normal',
+    override_mode = Cpt(EpicsSignalWithRBV, ':OVRD_ON', kind='normal',
                         doc='Epics Command to set Override mode')
-    override_force_open = Cpt(EpicsSignalWithRBV, ':VFS_FORCE_OPN',
+    override_force_open = Cpt(EpicsSignalWithRBV, ':FORCE_OPN',
                               kind='normal',
-                              doc=('Epics Command to force open the valve in'
-                                   'override mode'))
+                              doc=('Epics Command to force open'
+                                   'the valve in override mode'))
+    gfs_name = Cpt(EpicsSignalRO, ':GFS_RBV', kind='normal',
+                   doc='Gauge Fast Sensor Name')
```

Comment:
This should be `string` otherwise it will present the widget as a waveform plot. Please fix this.

Reply:
`string=True` is the kwarg needed here

Reply:
Attempt to fix in #467